src/views/Game/Game.styl

```diff
@@ -35,6 +47,8 @@
     top: 0;
 }
 
+// Is this really intended as a global scope css directive?
+// I don't dare scope it for now, but it looks odd: if it's global, it should probably be in ogs.styl?
```

Comment:
Looks like pretty old code, I think it's only used in this view and should be properly scoped